strr-host-pm-web/tests/unit/application-dashboard.spec.ts

```diff
@@ -174,4 +174,23 @@ describe('Dashboard Application Page', () => {
     expect(documentsList.findAll('[data-test-id="stored-document"]').length).toBe(3) // should show additional doc
     expect(documentsList.findAllComponents(UBadge).length).toBe(1) // should show date badge for NOC doc
   })
+
+  it('renders dashboard with No CRA Tax Number', () => {
+    vi.mock('@/stores/hostOwner', () => ({
+      useHostOwnerStore: () => ({
+        validateOwners: () => true,
+        hostOwners: [{
+          ...mockHostOwner,
+          taxNumber: '' // no tax number owner
```

Comment:
Should `isCraNumberOptional` be set as true here?

Reply:
`isCraNumberOptional` is not in the interface, only on that page.